src/main/java/com/ksm/wstbackoffice/domain/Warehouse.java

```diff
@@ -0,0 +1,27 @@
+package com.ksm.wstbackoffice.domain;
```

Comment:
Please rename package from domain to entity

Reply:
done